versioned_docs/version-21.10/getting-started/mysql_tuto.md

```diff
@@ -0,0 +1,72 @@
+---
+id: monitor-mysql-server
+title: Monitor a MySQL or MariaDB database
+---
+
+## Prerequisites
+
+The objective of this tutorial is to monitor a Mysql/MariaDB database.
+
+In order to be able to collect the necessary information from the database, a database user with specific privileges is required. If the database you want to monitor is a Centreon database, a dedicated user already exists and is called **centreon**. Otherwise, start by creating a new user: log in to your database, then run the following command:
+
+```shell
+CREATE USER 'username'@'IP_POLLER' IDENTIFIED BY 'password';
+```
+
+Then, whether your database is a Centreon database or not, run the following command:
+
+```shell
+grant ALL PRIVILEGES on *.* to 'username'@'IP_POLLER' identified by 'password'
+```
```

Comment:
Please use GRANT SELECT and respect the CAPITAL letter :)
Something like this:
GRANT SELECT ON . TO 'centreon'@'pollerip';

Not necessary to precise again the password ;-)